docs/examples/classification.ipynb

```diff
@@ -0,0 +1,946 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Classification on Nursery Dataset\n",
+    "This notebook demonstrates model diagnostics on the nursersy dataset https://www.openml.org/d/43938.\n",
+    "\n",
+    "The modelling goal is to predict the expectation $E(Y|X)$ of \n",
+    "$$\n",
+    "Y\n",
+    "= \n",
+    "\\begin{cases}\n",
+    "1 \\text{ if class = \"priority\" }\\\\\n",
+    "0 \\text{ if otherwise }\n",
+    "\\end{cases}\n",
+    "$$\n",
+    "\n",
+    "conditional on the features $X$.\n",
+    "\n",
+    "The impatient can directly jump to chapters 3-4 where the capability of `model-diagnostics` are shown.\n",
+    "Chapters 1-2 are needed for data preparation and model training."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1. Data\n",
+    "### 1.1 Data Load and Preprocessing"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "from sklearn.datasets import fetch_openml\n",
+    "from sklearn.model_selection import train_test_split"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "X, y = fetch_openml(data_id=43938, as_frame=True, return_X_y=True, parser=\"auto\")\n",
+    "df_original = pd.concat([X, y], axis=1)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "The prepared dataset contains 12958 rows.\n"
+     ]
+    }
+   ],
+   "source": [
+    "df = df_original.query(\"`class` != 'recommend'\").copy()\n",
```

Comment:
Why this filter?

Reply:
We don't need it in case of binary classification. For multiclass classification, it is because only two rows have been labelled as 'recommend'